jarviscli/CmdInterpreter.py

```diff
@@ -243,6 +243,33 @@ def spinner_stop(self, message="Task executed successfully! ", color=Fore.GREEN)
 
     def is_spinner_running(self):
         return self.spinner_running
+    
+    def create_routine(self, commands, name):
```

Comment:
Does this function and ``execute_routine`` need to be implemented in ``CmdInterpreter.py``? Would prefer this code in ``routine.py``.